### PR TITLE
Freeman: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added



### LINKED ISSUE #3121: Add Freeman
Repo: https://github.com/rfuenzalida/Freeman

***Onboarding Process:***
- [x] Contact author
- [x] QA-assessment / report
- [x] Build
- [x] Fontbakery QA
- [x] Regression QA (for font updates)
- [x] Visual QA

```
New goal for the project 2:
Publishing the new version under a new name 'Freeman'
```

***Onboarding Process Part 2***
- [x] Change the name in the source file
- [x] Ask the Rodrigo for a new description file
- [x] Create new information files: Authors, Contributors, Readme
- [x] Include CI GHA
- [x] Fontbakery QA
- [x] Visual QA
- [x] Proofing html
- [x] Font Images
- [x] PR to upstream
- [x] PR to GF

### PR DESCRIPTION
**Pull request by vv-monsalve**
Taken from the upstream repo https://github.com/rfuenzalida/Freeman/master at commit https://github.com/rfuenzalida/Freeman/master/commit/5ced8a95f5b0e0c3dcea3b033398ba93a56fb687.

Resolves #3121
## PR Checklist:

- [x] Family categorization tags collected from the type design team with the Categories Form
~- [ ] Minisite_url definition in the METADATA.pb file for commissioned projects~
~- [ ] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target~
- [x] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [x] Diffenator2 regression checks for revisions on all projects that are currently in production
- [ ] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [x] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Hi @emmamarichal 
- 🔥 **FAIL** Check tabular widths don't have kerning.
    This appears to be a false Fail. The font doesn't have the reported kernings
- 🔥 **FAIL** Shapes languages in all GF glyphsets.
    Per private conversation it was agreed to publish the font as is for this v1.000
- Designer bio for Aoife will be added later

File: ofl/freeman/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Freeman Project Authors (https://github.com/rfuenzalida/Freeman)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
```
Comment (emmamarichal) [inline ofl/freeman/OFL.txt:5]: url to update

**Review by emmamarichal — CHANGES_REQUESTED** [review]

**Comment by emmamarichal** [comment]
@vv-monsalve 
`Designer bio for Aoife will be added later`
-> She already has one, so we are good for designers profiles :)

File: ofl/freeman/OFL.txt (hunk 1 above)
Comment (vv-monsalve) [reply ofl/freeman/OFL.txt]: Solved.

**Review by emmamarichal — APPROVED** [review]

**Pull request merged**